hit_lc_proxy.bash

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# This script allows running against a specific lantern-cloud proxy identified by the PFE IP
+# It assumes that lantern-cloud is located on the filesystem as a sibling of lantern-desktop
+
+set -euf -o pipefail
+
+PROXY="${1:?please specify the proxy IP}"
+TMPDIR="${TMP:-/tmp}/hit_lc_proxy/$PROXY"
+OUTFILE="$TMPDIR/proxies.yaml"
+
+rm -rf "$TMPDIR"
+mkdir -p "$TMPDIR"
+
+echo "Generating config for ${PROXY} in ${OUTFILE}..."
+../lantern-cloud/bin/ptool route dump-config --legacy "$PROXY" > "$OUTFILE"
```

Comment:
Just as a heads up, `ptool` is [going away soon](https://github.com/getlantern/lantern-cloud/pull/682), to be subsumed by `lantern-cloud/bin/lc`.  I'll try to keep track of this and update as necessary.

Reply:
Thanks for the heads up! Didn't know that. I can switch to `lantern-cloud/bin/lc` once it's fully deprecated